contrib/opencensus-ext-azure/README.rst

```diff
@@ -25,7 +25,9 @@ This example shows how to send a warning level log to Azure Monitor.
 
 * Create an Azure Monitor resource and get the instrumentation key, more information can be found `here <https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource>`_.
 * Put the instrumentation key in ``APPINSIGHTS_INSTRUMENTATIONKEY`` environment variable.
-* You can also specify the instrumentation key explicitly in the code, which will take priority over a set environment variable.
+* You can also utilize a connection string with an instrumentation key. Place the connetion string in ``APPLICATIONINSIGHTS_CONNECTION_STRING`` environment variable, which will take priority over the instrumentation key environment variable.
```

Comment:
If we have connection string, we should put it as the recommended approach (1st choice) and use it across all examples.